tests/integration/contracts/test_oracle_daemon_config.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from src.constants import TOTAL_BASIS_POINTS
+from tests.integration.contracts.contract_utils import check_contract, check_value_type
+
+
+@pytest.mark.integration
+def test_oracle_daemon_config_contract(oracle_daemon_config_contract, caplog):
+    check_contract(
+        oracle_daemon_config_contract,
+        [
+            ('normalized_cl_reward_per_epoch', None, lambda response: check_value_type(response, int)),
+            (
+                'normalized_cl_reward_mistake_rate_bp',
+                None,
+                lambda response: check_value_type(response, int) and response < TOTAL_BASIS_POINTS,
+            ),
+            (
+                'rebase_check_nearest_epoch_distance',
+                None,
+                lambda response: check_value_type(response, int) and response < TOTAL_BASIS_POINTS,
```

Comment:
Comparison with `TOTAL_BASIS_POINTS` is meaningless?

Reply:
Fixed